docs/sphinx/source/whatsnew/v0.7.2.rst

```diff
@@ -10,6 +10,12 @@ API Changes
 * :py:func:`pvlib.iotools.read_tmy3` when coerced to a single year now returns
   indices that are monotonically increasing. Therefore, the last index will be
   January 1, 00:00 of the *next* year. (:pull:`910`)
+* Renamed :py:mod:`pvlib.losses` to :py:mod:`pvlib.soiling`. Additional loss
+  models will go into code modules named for the loss or effect type.
+  (:issue:`935`, :issue:`891`)
+* Renamed :py:func:`pvlib.losses.soiling_hsu` to :py:func:`pvlib.soiling.hsu`
+  and :py:func:`pvlib.losses.soiling_kimber` to :py:func:`pvlib.soiling.kimber`
```

Comment:
ditto twice, I believe these refs will not render since there is no `pvlib.losses` in this version of the docs